base/test/mp4writersink_tests.cpp

```diff
@@ -219,100 +266,14 @@ BOOST_AUTO_TEST_CASE(setgetprops_jpeg)
-	Logger::setLogLevel(boost::log::trivial::severity_level::info);
-	Logger::initLogger(loggerProps);
-
-	auto fileReaderProps = FileReaderModuleProps(inFolderPath, 0, -1);
-	fileReaderProps.fps = 100;
-	fileReaderProps.readLoop = false;
-
-	auto fileReader = boost::shared_ptr<Module>(new FileReaderModule(fileReaderProps));
-	auto h264ImageMetadata = framemetadata_sp(new H264Metadata(width, height));
-	fileReader->addOutputPin(h264ImageMetadata);
-
-	auto mp4WriterSinkProps = Mp4WriterSinkProps(41, 1, 100, outFolderPath);
-	mp4WriterSinkProps.logHealth = true;
-	mp4WriterSinkProps.logHealthFrequency = 100;
-	auto mp4WriterSink = boost::shared_ptr<Module>(new Mp4WriterSink(mp4WriterSinkProps));
-	fileReader->setNext(mp4WriterSink);
-
-	// #Dec_27_Review - do manual init, step and use saveorcompare
-
-	boost::shared_ptr<PipeLine> p;
-	p = boost::shared_ptr<PipeLine>(new PipeLine("test"));
-	p->appendModule(fileReader);
-
-	if (!p->init())
-	{
-		throw AIPException(AIP_FATAL, "Engine Pipeline init failed. Check IPEngine Logs for more details.");
-	}
-
-	LOG_ERROR << "processing folder <" << inFolderPath << ">";
-	p->run_all_threaded();
-
-	Test_Utils::sleep_for_seconds(10);
-
-	p->stop();
-	p->term();
-	p->wait_for_all();
-	p.reset();
+	writeH264(false, 10, outFolderPath);
 }
 
 BOOST_AUTO_TEST_CASE(h264_to_mp4v_chunking)
 {
-	int width = 704;
-	int height = 576;
-
-	std::string inFolderPath = "./data/h264_data/";
 	std::string outFolderPath = "./data/testOutput/mp4_videos/h264_videos/";
-
-	LoggerProps loggerProps;
-	loggerProps.logLevel = boost::log::trivial::severity_level::info;
-	Logger::setLogLevel(boost::log::trivial::severity_level::info);
-	Logger::initLogger(loggerProps);
-
-	auto fileReaderProps = FileReaderModuleProps(inFolderPath, 0, -1);
-	fileReaderProps.fps = 100;
-	fileReaderProps.readLoop = true;
-
-	auto fileReader = boost::shared_ptr<Module>(new FileReaderModule(fileReaderProps));
-	auto h264ImageMetadata = framemetadata_sp(new H264Metadata(width, height));
-	fileReader->addOutputPin(h264ImageMetadata);
-
-	auto mp4WriterSinkProps = Mp4WriterSinkProps(1, 1, 100, outFolderPath);
-	mp4WriterSinkProps.logHealth = true;
-	mp4WriterSinkProps.logHealthFrequency = 100;
-	auto mp4WriterSink = boost::shared_ptr<Module>(new Mp4WriterSink(mp4WriterSinkProps));
-	fileReader->setNext(mp4WriterSink);
-
-	// #Dec_27_Review - do manual init, step and use saveorcompare
-
-	boost::shared_ptr<PipeLine> p;
-	p = boost::shared_ptr<PipeLine>(new PipeLine("test"));
-	p->appendModule(fileReader);
-
-	if (!p->init())
-	{
-		throw AIPException(AIP_FATAL, "Engine Pipeline init failed. Check IPEngine Logs for more details.");
-	}
-
-	LOG_ERROR << "processing folder <" << inFolderPath << ">";
-	p->run_all_threaded();
-
-	Test_Utils::sleep_for_seconds(130);
-
-	p->stop();
-	p->term();
-	p->wait_for_all();
-	p.reset();
+	writeH264(true, 130, outFolderPath);
 }
 
 BOOST_AUTO_TEST_CASE(h264_metadata, *boost::unit_test::disabled())
```

Comment:
Why disabled test ?

Reply:
I had made these tests disabled locally , forgot to enable it while raising the PR